src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
```

Comment:
Consider defining the number of iterations (3) as a constant. According to the checklist, 'All magic numbers in your code should be constants.' This will make your code more readable and maintainable.